frontend/src/app/supportAdmin/UnarchivePatients/UnarchivePatientContainer.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { useGetOrganizationsQuery } from "../../../generated/graphql";
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+import { showError } from "../../utils/srToast";
+
+import UnarchivePatients from "./UnarchivePatient";
+
+export type UnarchivePatientOrganization = {
+  internalId: string;
+  externalId: string;
+  name: string;
+  facilities?: {
+    id: string;
+    name: string;
+  }[];
+};
+
+export type UnarchivePatientFacility = {
+  id: string;
+  name: string;
+};
+
+export type UnarchivePatientPatient = {
+  internalId: string;
+  firstName: string;
+  lastName: string;
+  middleName: string;
+  birthDate: string;
+  isDeleted: boolean;
+  facility: UnarchivePatientFacility | null;
+};
+
+const getSortedOrgs = (orgs: UnarchivePatientOrganization[] | undefined) => {
+  return orgs ? orgs.sort((a, b) => (a.name > b.name ? 1 : -1)) : [];
+};
+
+const UnarchivePatientContainer = () => {
+  const { pageNumber } = useParams();
+  const currentPage = pageNumber ? +pageNumber : 1;
+
+  const { data, loading, error } = useGetOrganizationsQuery({
+    fetchPolicy: "no-cache",
+    variables: { identityVerified: true },
+  });
+  const sortedOrgs = getSortedOrgs(data?.organizations);
+  const [selectedOrgId, setSelectedOrgId] = useState<string>("");
+  const [selectedFacilityId, setSelectedFacilityId] = useState<string>("");
+  const [archivedPatientsCount, setArchivedPatientsCount] = useState<
+    number | undefined
+  >(undefined);
```

Comment:
⚠️ setting the patients count and patients default to be `undefined` to differentiate from when we get 0 results or no patients back (for UI purposes)